uwblueprint/sistering
# Create volunteer navbar

## Ticket link

Closes #125 



## Implementation description
* Created `VolunteerNavbar.tsx` component in `/frontend/src/components/volunteer`
* Navbar tabs switch between `VolunteerShiftsPage.tsx` and the `VolunteerPostingsPage.tsx`
* Added navbar to those two pages



## Steps to test
1. Navigate to either `/volunteer/shifts` or `/volunteer/postings` route where the navbar is used
2. Check that the navbar tabs switch to the correct pages



## What should reviewers focus on?
* Navbar follows desgins
* Navbar correctly switches between the two pages


## Checklist
